app/assets/stylesheets/application.css.scss

```diff
@@ -93,6 +93,12 @@ a, a:visited, a:link {
     text-align: center;
 }
 
+.flash_remove {
+  color: green;
```

Comment:
Color `green` should be written in hexadecimal form as `#008000`<br>Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.